src/classes/frSetupController.cls

```diff
@@ -5,6 +5,27 @@ public with sharing class frSetupController {
 		}
 		private set;
 	}
+	public static String PERSON_ACCOUNT_TYPE {
+		get {
+			//TODO Put it in the class, maybe a new field on frField to diff between the 2?
```

Comment:
Leaving a comment here to keep track of this TODO for refactoring the types (if we decide to)